modules/network-observability-lokistack-create.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-lokistack-create_{context}"]
+
+= Create a LokiStack custom resource 
+It is recommended to deploy the LokiStack in the same namespace referenced by the FlowCollector specification, `spec.namespace`. You can use the web console or CLI to create a namespace, or new project. 
+
+.Procedure
+
+. Navigate to *Operators* -> *Installed Operators*. In the Details, under *Provided APIs*, select `LokiStack` and click *Create LokiStack*. 
```

Comment:
It seems like the second sentence should be a separate step.

Reply:
ACK, agree w/ this observation